app/node.ts

```diff
@@ -122,6 +102,11 @@ async function main() {
     console.error(err);
     process.exit();
   }
+
+  // handle "ctrl + c" as a graceful exit - tested on Linux 20191205
+  process.on("SIGINT", async function() {
```

Comment:
Nice way to handle the exists! I did not know we could do this.

Reply:
It was totally Sean’s idea to use an event handler :-)  I did some research thinking that it was going to be harder but was shocked at how elegant/easy it was.